curriculum/challenges/english/02-javascript-algorithms-and-data-structures/object-oriented-programming/add-methods-after-inheritance.md

```diff
@@ -58,10 +58,10 @@ assert(typeof Animal.prototype.bark == 'undefined');
 assert(typeof Dog.prototype.eat == 'function');
 ```
 
-`Dog` should have the `bark()` method as an `own` property.
+`Dog` prototype should have a `bark()` method.
```

Comment:
```suggestion
The `Dog` prototype should have a `bark()` method.
```

Since we're referring to the prototype now, I think we need `the`.

Reply:
Yup you're right, I'll add it in now.